rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -70,6 +70,9 @@ create_node(
     participant_qos.user_data.value[sizeof(prefix) - 1 + name_length] = ';';
   }
 
+  // Setting the contentfilter_property_max_length to 1024
```

Comment:
Can you please modify the comment to describe why it's needed and how 1024 has been picked?

Reply:
Yes, I'm still investigating the exact number that should be used for the property, will update once I have a reasonable one.

Reply:
for reference: https://community.rti.com/static/documentation/connext-dds/5.3.1/doc/api/connext_dds/api_cpp/structDDS__DomainParticipantResourceLimitsQosPolicy.html#aef1ff851202f5777d31528a52d528e4f

From the RTPS spec:
```
ContentFilterProperty_t: Mapping of the type

typedef string<256> String256;
typedef sequence<string> StringSequence;
struct ContentFilterProperty_t {
  String256 contentFilteredTopicName;
  String256 relatedTopicName;
  String256 filterClassName;
  string filterExpression;
  StringSequence expressionParameters;
};
```

Reply:
yes I think it makes sense to have the link in the comment as well